executor/builder.go

```diff
@@ -1242,24 +1243,36 @@ func (b *executorBuilder) getStartTS() (uint64, error) {
 
 func (b *executorBuilder) buildMemTable(v *plannercore.PhysicalMemTable) Executor {
 	var e Executor
-	switch v.Table.Name.L {
-	case strings.ToLower(infoschema.TableClusterConfig):
-		e = &ClusterReaderExec{
-			baseExecutor: newBaseExecutor(b.ctx, v.Schema(), v.ExplainID()),
-			retriever: &clusterConfigRetriever{
-				extractor: v.Extractor.(*plannercore.ClusterConfigTableExtractor),
-			},
+	if util.IsMemOrSysDB(v.DBName.L) {
+		switch v.DBName.L {
+		case util.MetricSchemaLowerName:
+			e = &MetricReaderExec{
+				baseExecutor: newBaseExecutor(b.ctx, v.Schema(), v.ExplainID()),
+				table:        v.Table,
+				outputCols:   v.Columns,
+			}
 		}
-	default:
-		tb, _ := b.is.TableByID(v.Table.ID)
-		e = &TableScanExec{
-			baseExecutor:   newBaseExecutor(b.ctx, v.Schema(), v.ExplainID()),
-			t:              tb,
-			columns:        v.Columns,
-			seekHandle:     math.MinInt64,
-			isVirtualTable: !tb.Type().IsNormalTable(),
+		switch v.Table.Name.L {
+		case strings.ToLower(infoschema.TableClusterConfig):
```

Comment:
It's not good, table name `cluster_config` can also exist in other system databases in theory. 
I suggest remove `if util.IsMemOrSysDB(v.DBName.L)`,  and check both schema name and table name for each table.

Reply:
great, done.